### ISSUE TITLE
multiple renderer windows

### ISSUE THREAD

**Issue by farfromrefug** [electron-userland/electron-webpack#134]
Hi,

My app have multiple renderer windows with different html/js files. All inside the renderer folder.
How can i add multiple entries for them?

**Comment by mhuggins** [comment]
I'm interested in understanding how to do this as well.

**Comment by fabiospampinato** [comment]
I've published this very simple router for solving the problem: https://www.npmjs.com/package/react-router-static, just add a `?route=***` parameter to the url you're set.

**Comment by febox6** [comment]
@fabiospampinato my project does not use React, how to achieve the same without it?

**Comment by fabiospampinato** [comment]
@febox6 the question is so general that I'd answer it with: learn to program.

**Comment by febox6** [comment]
Ok, thanks anyway, from your answer I got that the router solution is not react exclusive.

**Comment by fabiospampinato** [comment]
@febox6 yeah "routing" just means taking a different path depending on whatever. No matter the UI framework you're using you can always do this:

- From the main process tell the renderer process which route/path to take, in my example above I'm doing that my setting a url parameter called "route".
- Then from the renderer process you read that parameter and do something different (in this case load the UI for a different window) depending on what it says.
